packages/lesswrong/components/posts/PostsItem2.tsx

```diff
@@ -35,8 +35,7 @@ export const styles = (theme) => ({
   postsItem: {
     display: "flex",
     position: "relative",
-    paddingTop: 10,
-    paddingBottom: 10,
+    height: 46,
```

Comment:
Alas, this line broke the whole mobile layout. Note to check mobile more frequently, or to be more suspicious of changes of this type.